sources/north-america/us/in/IndianaMap_Latest.geojson

```diff
@@ -11,7 +11,7 @@
         "icon": "https://www.arcgis.com/sharing/rest/content/items/c470d38a730b4a5d839602174ed97d5c/resources/ingov-logo-white.png",
         "url": "https://imagery.gis.in.gov/arcgis/rest/services/Imagery/BestAvailableTiles/ImageServer/exportImage?f=image&format=jpg&bbox={bbox}&imageSR={wkid}&bboxSR={wkid}&size={width},{height}&foo={proj}",
         "max_zoom": 19,
-        "license_url": "https://gis.iu.edu/dataset/statewide/in_2016.html",
+        "license_url": "https://www.in.gov/gis/files/Sanborn-Ortho-Presentation-1_8_2021_.pdf",
```

Comment:
This slide deck just says the imagery is “[free of charge](https://www.in.gov/gis/files/Sanborn-Ortho-Presentation-1_8_2021_.pdf#page=9)”.

The original statement releasing the 2016–19 Woolpert imagery into the public domain has been [archived by the Wayback Machine](http://web.archive.org/web/20220125235228/https://gis.iu.edu/dataset/statewide/in_2016.html) and should still be valid.

The 2020–23 project is also in the public domain, based on [this request for proposals](https://www.in.gov/idoa/proc/bids/RFP-20-051/051_Att%20H%20-%20Summary%20Scope%20of%20Work.docx) that the state awarded to Sanborn. Page 2, section 1.4.1:

> All orthoimagery and ancillary data products delivered through this contract shall be public domain data without restrictions on distribution or use.  Ownership of the final data products and any ancillary products will reside with the State of Indiana.

Reply:
I just copy-pasted the [url used for the 2021 service](https://github.com/osmlab/editor-layer-index/commit/4014bd5969693a6e196be208be6087c5dca5f370). Since this will include both 2017-2019 and 2021, not sure which one to add.

Reply:
I guess the 2020–23 license is more relevant, since the Sanborn imagery will eventually replace the older Woolpert imagery in this layer.